libs/sdk-embedding/src/iframe/common.ts

```diff
@@ -2,6 +2,7 @@
 import isEmpty from "lodash/isEmpty";
 import isObject from "lodash/isObject";
 import { GdcExecuteAFM } from "@gooddata/api-model-bear";
+import * as Model from "@gooddata/sdk-model";
```

Comment:
Please avoid `import * as` style imports, this will hurt tree-shaking and make the budles bigger than necessary.

Reply:
ok, will use local renames of imports